server/db/models/user.js

```diff
@@ -3,12 +3,31 @@ const Sequelize = require('sequelize')
 const db = require('../db')
 
 const User = db.define('user', {
+  name: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
+  img: {
+    type: Sequelize.STRING,
+    allowNull: false,
+  },
   email: {
     type: Sequelize.STRING,
     unique: true,
-    allowNull: false
+    allowNull: false,
+    isEmail: true
   },
   password: {
+    type: Sequelize.STRING,
+    allowNull: false
+  },
+  rating: {
+    type: Sequelize.INTEGER
```

Comment:
Should be a calculation, like an average of their transaction ratings where they are the tutor